webgrid/__init__.py

```diff
@@ -567,22 +532,18 @@ def render_html(self, record, hah):
             return data
         return self._format_datetime(data, self.html_format)
 
-    def render_xls(self, record):
+    def render_xlsx(self, record):
         data = self.extract_and_format_data(record)
         if not data:
             return data
         # if we have an arrow date, pull the underlying datetime, else the renderer won't know
         #   how to handle it
         if arrow and isinstance(data, arrow.Arrow):
             data = data.datetime
-        # xlwt has no idea what to do with zone information
         if isinstance(data, dt.datetime) and data.tzinfo is not None:
```

Comment:
Would be worth testing whether xlsxwriter and openpyxl can handle the timezone info, and if so, remove this clause.

Reply:
I tested just with xlsxwriter without this clause and got `TypeError: Excel doesn't support timezones in datetimes. Set the tzinfo in the datetime/time object to None or use the 'remove_timezone' Workbook() option`

Reply:
In that case, instead of removing the comment, I'd change it to indicate that Excel, not xlwt, is the layer that doesn't process the zone.

Reply:
Done